glide.yaml

```diff
@@ -0,0 +1,40 @@
+package: github.com/projectcalico/app-policy
+import:
+- package: github.com/docopt/docopt-go
+  version: ^0.6.2
+- package: github.com/golang/protobuf
+  version: master
+  subpackages:
+  - proto
+- package: github.com/projectcalico/libcalico-go
+  version: master
```

Comment:
after some discussion, we (tigera devs) decided to use a version or a hash instead of a branch name since if you sue a branch name and once it's cached locally it will use that instead of querying upstream. If you wan't to use the latest code, you can do that by not having the `version` line at all, glide will fetch the latest code every time if the version is not specified. That's better than `master` IMO

Reply:
+1